integration-tests/jest/jest-utils.js

```diff
@@ -1,7 +1,9 @@
 const _ = require('lodash/fp')
 const { defaultTimeout } = require('../utils/integration-helpers')
 const { withScreenshot, withPageLogging } = require('../utils/integration-utils')
+const { Cluster } = require('puppeteer-cluster')
 const envs = require('../utils/terra-envs')
+const rawConsole = require('console')
```

Comment:
I see `rawConsole` defined like above and also usages of just `console.log` in the codebase. Why is that?

Reply:
Using regular `console` within jest generates a lot of noise. It wraps the statements in some newlines and the line number if the console. `rawConsole` bypasses this, simply printing the message